pkg/k8shandler/configuration_tmpl.go

```diff
@@ -15,7 +15,7 @@ node:
   max_local_storage_nodes: 1
 
 network:
-  host: 0.0.0.0
+  host: _local_
```

Comment:
seems like these are two different things
`0.0.0.0` means bind on any address
`_local_` to me means only local access is allowed... 
can you confirm?

Reply:
> seems like these are two different things
> `0.0.0.0` means bind on any address
> `_local_` to me means only local access is allowed...
> can you confirm?

```
_local_ | Any loopback addresses on the system, for example 127.0.0.1.
```
* https://www.elastic.co/guide/en/elasticsearch/reference/6.8/modules-network.html#network-interface-values
*https://www.elastic.co/guide/en/elasticsearch/reference/6.8/network.host.html

Reply:
yeah, this is not what we want then. that means only access to ES would be from itself

Reply:
reverted.  We'll have to address if it fails ipv6

Reply:
why not make this `_global_` ?